### DISCUSSION TITLE
Sigv4 Release Date

### DISCUSSION THREAD

**Discussion by decipher27** [prometheus-operator/prometheus-operator#4339] [Q&A] [answered]
Could you let me know when you're planning to release Sig v4? So, that the prometheus server will remote_write to Amazon Prometheus. 
- Thank you

**Comment by simonpasquier** [comment, marked as answer]
we have a public [release schedule](https://github.com/prometheus-operator/prometheus-operator/blob/master/RELEASE.md#release-schedule) and v0.52 is planned for early november.

**Reply by andrewgkew** [reply]
@simonpasquier - I read somewhere 0.52 is being released today?

**Reply by andrewgkew** [reply]
@paulfantom mentioned this on 20th October: https://github.com/prometheus-operator/prometheus-operator/pull/3994#issuecomment-947511460

**Reply by paulfantom** [reply]
It is released.
